### ISSUE TITLE
Wiki Page - Guide for Lawyers

### ISSUE THREAD

**Issue by passamo9** [passamo9/LaTeX_EnglandWales_Templates#20]

**Comment by passamo9** [comment]
Thanks for closing #21!

**Comment by Giebisch** [comment]
I just wanted to mention that currently @passamo9 you are the only who can add & edit wiki pages. If this is what you want then it's fine, but we are unable to help you out then.

**Comment by Andonome** [comment]
I've added individual setup instructions to my wiki.

If you'd like to check them out, clone the wiki locally:

 > git clone https://github.com/passamo9/LaTeX_EnglandWales_Templates.wiki.git

Then add mine as a remote:

> git remote add andonome https://github.com/Andonome/LaTeX_EnglandWales_Templates.wiki.git

Make a new branch to check the differences

> git switch -c setup

> git diff master

Some screenshots of a Mac should probably go in there before merging with the master branch.

**Comment by passamo9** [comment]
> > git clone https://github.com/passamo9/LaTeX_EnglandWales_Templates.wiki.git

This unfortunately was 404 for me. I'm probably doing something wrong.

>>git remote add andonome https://github.com/Andonome/LaTeX_EnglandWales_Templates.wiki.git

Same thing. What am I doing wrong?

>Make a new branch to check the differences

I will do this next


>Some screenshots of a Mac should probably go in there before merging with the master branch.

Ok I will do this tomorrow!

**Comment by yjyao** [comment]
@Andonome did you try creating a PR on the wiki repo? i guess it didn't work? what does it say?

@passamo9 those were commands you needed to run. not links for clicking (@Andonome should have used code format instead of quotes tho). i don't think @passamo9 has any experience working with [command line](https://https://en.wikipedia.org/wiki/Command-line_interface) so this is probably going to be hard for them.

i seem to remember @Andonome you posted your wiki page as a comment somewhere? what was the problem with that approach?

**Comment by passamo9** [comment]
>i don't think @passamo9 has any experience working with command line so this is probably going to be hard for them.

Sadly I don't. Would it be possible for you to do this? I really appreciate your help.

**Comment by yjyao** [comment]
well i can but the point here is to get you to review it i think. for now you can head over to https://github.com/Andonome/LaTeX_EnglandWales_Templates/wiki to check @Andonome's wiki out. and then you can review here.

@passamo9 do you know how to open a terminal (if you don't know what that is, you probably don't know how)?

**Comment by passamo9** [comment]
>well i can but the point here is to get you to review it i think. for now you can head over to https://github.com/Andonome/LaTeX_EnglandWales_Templates/wiki to check @Andonome's wiki out.

That looks great! We should add this.

I hasten to add that the chance that a lawyer has Ubuntu installed on their computer is infinitesimally small. But for the sake of completeness, this is a great idea!

>@passamo9 do you know how to open a terminal (if you don't know what that is, you probably don't know how)?

I opened the terminal on my MacBook once, but not here on GitHub.

**Comment by Andonome** [comment]
>  did you try creating a PR on the wiki repo? i guess it didn't work? what does it say?

I didn't know wiki's allowed that. I'll check it out next time.

**Comment by yjyao** [comment]
> @Andonome I didn't know wiki's allowed that. I'll check it out next time.

no i don't know if it'll work that's why i was curious.

> @passamo9 I opened the terminal on my MacBook once, but not here on GitHub.

right you should open the terminal locally on your machine. not here on GitHub. if that's the case you can try the commands out, just to get a flavor of it. in sequence, paste _each line_ of command (everything after `$`) and hit enter:

```shell
$ git clone https://github.com/passamo9/LaTeX_EnglandWales_Templates.wiki.git
$ cd LaTeX_EnglandWales_Templates.wiki
$ git remote add andonome https://github.com/Andonome/LaTeX_EnglandWales_Templates.wiki.git
$ git diff origin/master andonome/master
```

then use `k`/`j` keys to move up/down, and finally press `q` to quit.

**Comment by passamo9** [comment]
![Uploading image.png…]()

Did I fuck something up?

**Comment by passamo9** [comment]
Did I fuck something up?

<img width="634" alt="image" src="https://user-images.githubusercontent.com/69139574/108037685-3c7a6380-703a-11eb-9f88-2079d3886881.png">

**Comment by Andonome** [comment]
`master` should be `main`.

'Master' is a long-stading default, but in your case that should be 

`git diff origin/main andonome/main`

**Comment by Andonome** [comment]
That'll show you the differences in the 'main' branches, but to see what's different in the `gdpr` branch, it would be

`git diff origin/main andonome/gdpr`

**Comment by passamo9** [comment]
@Andonome I promise I will get back to this later

**Comment by Andonome** [comment]
No worries. I need to start my boring job now anyway.

**Comment by yjyao** [comment]
> `master` should be `main`.

no. it's `master`. `main` is the name of the main branch for the real repo. this is the wiki repo which has a default main branch called `master`. (actually i've been wondering why the main branch for this repo was called `main`. @passamo9 did you manually choose it? just curious)

note it complained `andonome/master` was not found but not `origin/master`. in fact, @passamo9 if you open the terminal and run

```shell
$ cd LaTeX_EnglandWales_Templates.wiki
$ git branch
```

you should see the branch name `master` listed.

if this is true, then proceed with downloading Andonome's `master` branch so you can diff against it. it should look like this (note you should _only_ copy-paste the line with `$`, that is: `git fetch andonome master`. the rest of it is what you should see):

```shell
$ git fetch andonome master
remote: Enumerating objects: 11, done.
remote: Counting objects: 100% (11/11), done.
remote: Compressing objects: 100% (6/6), done.
remote: Total 9 (delta 3), reused 9 (delta 3), pack-reused 0
Unpacking objects: 100% (9/9), done.
From https://github.com/Andonome/LaTeX_EnglandWales_Templates.wiki
 * branch            master     -> FETCH_HEAD
 * [new branch]      master     -> andonome/master
```

then you run the diff:

```tex
$ git diff origin/master andonome/master
```

and you should see stuff (presumably with red and green text) spitting out.
